app/api/helpers/files.py

```diff
@@ -160,3 +161,13 @@ def create_save_image_sizes(image_file, image_sizes_type, unique_identifier=None
     }
 
     return new_images
+
+
+def make_fe_url(settings=None, path=None):
```

Comment:
can you make this func name more descriptive, also add some more info in the docstring